### PR TITLE
test(core): load mime.types from java + file

### PR DESCRIPTION
**Pull request by bluestreak01**
ServerMain may not start if "mime.types" is on classpath but inside a jar. This could happen in tests.

### CODE REVIEW

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 27 / 29 (93.10%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/cutlass/http/MimeTypesCache.java|23|25|92.00%|
|:large_blue_circle:|io/questdb/cutlass/http/DefaultHttpServerConfiguration.java|4|4|100.00%|

File: core/src/main/java/io/questdb/cutlass/http/MimeTypesCache.java [hunk 1]
```diff
@@ -27,9 +27,39 @@
...
 import io.questdb.std.str.Path;
+import org.jetbrains.annotations.TestOnly;
+
+import java.io.*;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
 
 public final class MimeTypesCache extends CharSequenceObjHashMap<CharSequence> {
 
+    @TestOnly
```
Comment (puzpuzpuz) [inline core/src/main/java/io/questdb/cutlass/http/MimeTypesCache.java:38]: nit: technically this constructor is not test-only as it's used in `DefaultHttpServerConfiguration`

**Review by puzpuzpuz — APPROVED** [review]

File: core/src/main/java/io/questdb/cutlass/http/MimeTypesCache.java (hunk 1 above)
Comment (bluestreak01) [reply core/src/main/java/io/questdb/cutlass/http/MimeTypesCache.java:38]: damn :) lets remove this later. I really am looking forward to CI completion!

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 25 / 29 (86.21%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/cutlass/http/DefaultHttpServerConfiguration.java|3|5|60.00%|
|:large_blue_circle:|io/questdb/cutlass/http/MimeTypesCache.java|22|24|91.67%|

**Pull request merged**